nucliadb/tests/search/integration/api/v1/test_ask.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (C) 2021 Bosutech XXI S.L.
```

Comment:
Could you rename it to `test_ask_audit` or similar? So it's easier to find later?